### DISCUSSION TITLE
GCal Inline Embed

### DISCUSSION THREAD

**Discussion by wigglemuch** [Vaccarini-Lorenzo/iCalObsidianSync#31] [General]
Does this work with Google or only iCal? The code syntax to embed the calendar isn't working for me with my GCal. LOVE this brilliant plugin.

**Comment by Vaccarini-Lorenzo** [comment]
Hi @wigglemuch, I'm glad you like it! Yeah, the inline event view is designed for Google Calendar too. If you found any problem you can open an issue providing some more info and I'll work on it as soon as possible. Thank you!

**Reply by wigglemuch** [reply]
Just did! I'm probably just doing something wrong, thanks for the speedy reply. :)
